src/app/components/home/home.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss']
+})
+export class HomeComponent {
+  title: string = 'Card for Organizing Activities';
```

Comment:
this line and below shouldn't be in ts file. Instead you should pass the data in the html template of the home component.
For example:
<action-card
[cardTitle]="'My Sample Component Header'"
// other inputs go here
>
</action-card>